scio-core/src/main/scala/com/spotify/scio/values/SCollection.scala

```diff
@@ -1140,7 +1144,9 @@ sealed trait SCollection[T] extends PCollectionWrapper[T] {
    * @group output
    */
   def saveAsTextFile(path: String,
-                     suffix: String = ".txt", numShards: Int = 0): Future[Tap[String]] = {
+                     suffix: String = ".txt", numShards: Int = 0,
+                     header: Option[String] = None,
```

Comment:
Use `String` and `null` instead of `Option` for both `header` and `footer`.

Reply:
Sure! I generally try to avoid nulls in scala. I am a bit lost here, how can I put `@Nullable` here in scala?

Reply:
@labianchin you would do `@Nullable header: String = null`, actually, let's not do it here.